src/components/organisms/ChatCreatorForm/ChatCreatorForm.jsx

```diff
@@ -0,0 +1,85 @@
+import { Field, Formik } from 'formik';
+import { useNavigate } from 'react-router';
+import { Button, Text } from '../..';
+import { useCreateChatRoom, useUser } from '../../../hooks';
+import { routes } from '../../../routes/Routes';
+
+import './ChatCreatorForm.scss';
+import '../Form/Form.scss';
+
+export const ChatCreatorForm = () => {
+    const user = useUser();
+    const userID = user.data.uid;
+    const createChatRoom = useCreateChatRoom();
+
+    const navigate = useNavigate();
+    const redirectToChatRoom = (id) => navigate(`${routes.Chat}/${id}`);
+
+    return (
+        <div className='organism__chat_room_creator'>
+            <Formik
```

Comment:
We have Form component, couldn't we use that? We actually should

Reply:
It doesn't work with multiple fields